### ISSUE TITLE
PSR2.Methods.FunctionCallSignature multi line sniff breaks on multi-line concatenation

### ISSUE THREAD

**Issue by spadusa** [squizlabs/PHP_CodeSniffer#679]
PHPCS 2.3.3 (stable) in use, with a modified PSR2 standard:
```
<?xml version="1.0"?>
<ruleset name="eLYKModifiedPSR2">
 <description>eLYK's Modified PSR2 standard.</description>
 <rule ref="PSR2">
  <exclude name="Squiz.ControlStructures.ControlSignature.NewlineAfterOpenBrace"/>
  <exclude name="PSR1.Classes.ClassDeclaration.MultipleClasses"/>
  <exclude name="PSR1.Classes.ClassDeclaration.MissingNamespace"/>
  
 </rule>
 
  <rule ref="Squiz.Strings.DoubleQuoteUsage.ContainsVar" />
  <rule ref="Squiz.Strings.DoubleQuoteUsage.NotRequired" />
</ruleset>
```
```
wp_enqueue_script('tinysort-js', get_template_directory_uri() . //This line does not trigger a ContentAfterOpenBracket error
        '/js/tinysort.js', array('jquery'), '1.0.0', true); //This line does not trigger a MultipleArguments error
```

```
wp_enqueue_script('tinysort-js', get_template_directory_uri() . '/js/tinysort.js' //This line DOES trigger a ContentAfterOpenBracket error
array('jquery'), '1.0.0', true); //This line DOES trigger a MultipleArguments error
```

In summary, if the string concatenation spans multiple lines, the sniff breaks. I found this error because I was on 2.0.0RC2 (beta) and the sniff was working for me, but my coworker on 2.3.3 (stable) was not getting the error. When I upgraded to 2.3.3, I stopped getting the error.

I believe this should work the way it did on 2.0.0RC2 (beta) and not the way it currently works; the function is indeed spanning multiple lines, so the sniff should trigger and throw the error.

__EDIT__: Added ruleset.xml

**Comment by gsherwood** [comment]
PHP-FIG decided that a string concatenation across multiple lines was to be ignored. As are arrays, closures, and a few other things I've probably forgotten.

This was fixed as bug #412 and released in version 2.2.0.
